ngraph/core/src/op/deformable_convolution.cpp

```diff
@@ -195,6 +196,136 @@ std::shared_ptr<Node>
+                                         const CoordinateDiff& pads_end,
+                                         const ngraph::op::PadType& auto_pad,
+                                         const int64_t group,
+                                         const int64_t deformable_group,
+                                         const bool use_bilinear_interpolation_padding)
+    {
+        bool rc = true;
+        switch (inputs[0]->get_element_type())
+        {
+            NGRAPH_TYPE_CASE(evaluate_deformable_convolution,
+                             f32,
+                             inputs,
+                             out,
+                             strides,
+                             pads_begin,
+                             pads_end,
+                             dilations,
+                             auto_pad,
+                             group,
+                             deformable_group,
+                             use_bilinear_interpolation_padding);
+            NGRAPH_TYPE_CASE(evaluate_deformable_convolution,
+                             f16,
+                             inputs,
+                             out,
+                             strides,
+                             pads_begin,
+                             pads_end,
+                             dilations,
+                             auto_pad,
+                             group,
+                             deformable_group,
+                             use_bilinear_interpolation_padding);
+        default: rc = false; break;
+        }
+        return rc;
+    }
+} // namespace deformable_convolution
+
+bool op::v8::DeformableConvolution::evaluate(const HostTensorVector& outputs,
+                                             const HostTensorVector& inputs) const
+{
+    NGRAPH_OP_SCOPE(DeformableConvolution_v8_evaluate);
+    deformable_convolution::evaluate_deformable_convolution(inputs,
+                                                            outputs[0],
+                                                            get_strides(),
+                                                            get_dilations(),
+                                                            get_pads_begin(),
+                                                            get_pads_end(),
+                                                            get_auto_pad(),
+                                                            get_group(),
+                                                            get_deformable_group(),
+                                                            get_bilinear_interpolation_pad());
+    return true;
+}
+
+bool op::v8::DeformableConvolution::has_evaluate() const
+{
+    NGRAPH_OP_SCOPE(DeformableConvolution_v8_has_evaluate);
+    switch (get_input_element_type(0))
+    {
+    case ngraph::element::f16:
+    case ngraph::element::f32: return true;
+    default: break;
+    }
+    return false;
```

Comment:
looks like not covered by tests

Reply:
"evaluate" method will be deleted after merge cpu plugin implementation
this method has been implemented for testing purposes, e.g. launching models from paddlepaddle

Reply:
Do we have a deadline where CPU will have native support of deformable convolution?

Reply:
I can't say for sure what work week number, it has shifted somewhat, until FF of 2022.1 Release :)